src/commands/transaction/mosaicaddressrestriction.ts

```diff
@@ -0,0 +1,147 @@
+import chalk from 'chalk';
+import { command, ExpectedError, metadata, option} from 'clime';
+import {
+    Account,
+    Address,
+    Deadline,
+    MosaicAddressRestrictionTransaction,
+    MosaicId,
+    TransactionHttp,
+    UInt64,
+} from 'nem2-sdk';
+import {OptionsResolver} from '../../options-resolver';
+import {ProfileCommand, ProfileOptions} from '../../profile.command';
+import {MosaicRestrictionType} from '../../validators/mosaic.validator';
+
+export class CommandOptions extends ProfileOptions {
+    public static limitType = [0, 1, 2, 3, 4, 5, 6];
```

Comment:
It's a good first step. Although it was not specified, could we change it so that the user introduces  EQ, LT, GT instead of the number? Thanks

Reply:
I've changed it with EQ, LT, GT....
But I don't know how to use the property `newRestrictionType` when I set a mosaic address restriction.
@dgarcia360

Reply:
In mosaic address restriction transaction, it seems that this property is not used.